sdk/iot/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Track 2 libraries are using repo paths of sdk/<service>/<package>/.  
https://azure.github.io/azure-sdk/policies_repostructure.html

Reply:
Then we should name it sdks/iot/core I believe.